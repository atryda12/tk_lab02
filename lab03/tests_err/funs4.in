
int a = 0, b = 0, c = 0;

int trib(int n) {
    return nbonacci(n,3);
}

print trib(10);

