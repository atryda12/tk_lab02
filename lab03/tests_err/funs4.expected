
Error: Call of undefined fun: 'nbonacci': line 5
